src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/recordings/test_create_cluster_with_separate_kv.yaml

```diff
@@ -0,0 +1,2893 @@
2O8m4k+IA/6L7YZga/Jjb2SvPAAyi2LxK36U76NeaFwe6ZIqDrphFA71DrzlpUZ4a9nF/bgUVvsyse4WDk1pkOUkYeH1XFIc1POb0bcOgxBITrhQcQs9sdwO27uHS/aPsByTP8tj/IQ73ngVEzCw7GmKKFdI2tIQeeW753PBJM/XFbMPCDkO6WYDEf4nIE8IDsSCb7XopxOwq11I1XA6SydzUNuX1ZFOJ0CnLSfNZfovfQAG/Hcjczui4H+eFRLlDKoH4fHxG/W1Cj2G6WXfnfr2kdgyIAfXX16fzmPmh79NyU5fhZXTri1f6nrZwm+7JVIXC+/nMbLHb8z5UZufCA94xzhUMFFCrXESQbHS4aRcw2FUaw45jOPCYXezR8So2hIPNis3pmay9AMvQ4RTe8fU6BiKGYPSOT7Fsi63ObDxOvxbY/DYY5ebMO6rfQxgRC6jCtgaouAvNWni7SJTLHW4GxVamqBh9hZlDdcbq7ftrC3CrRaIcAMxW/zm8w/MxWDrbFJf9vDDaUWydkinlt7w3TtqBJVftcrEyH5aDsvf2QXV2CMyGO6xhdErJE1bMbBs8kip6OPU3TT+zrDG4ACQEBkdKIfDxVhov03keJ10ZSL8DBbynEoK5vqw9p6dPNdIKF/7NXGyW3hT74+2X94IFs1wcvm/orTr95tYPwf7GyrGfNXJGQt78Pkkxs0YPz9wyNAHPjo2ytPZClbJmF0f18tnB6l1EALBH9TrI+2vZ7Dm/FLMKyJVzY3OeSxj+NL7hzDvK/b0hyxqhEvJgIhasPqdrdiYtPqdkuYmZXHclNffDxJUgGo8eg6hIYrCswIm1kWEoo/xAqq1PDppCAUM2+pG6Goh/R84qQlKjGB6TATBgkqhkiG9w0BCRUxBgQEAQAAADBXBgkqhkiG9w0BCRQxSh5IADcANgA0ADQANABkAGUANgAtADMAOAA0ADYALQA0AGYANwAzAC0AOQBkADUANgAtADIANAA4ADIAYgA2ADYAOQBmADQAMgA3MHkGCSsGAQQBgjcRATFsHmoATQBpAGMAcgBvAHMAbwBmAHQAIABFAG4AaABhAG4AYwBlAGQAIABSAFMAQQAgAGEAbgBkACAAQQBFAFMAIABDAHIAeQBwAHQAbwBnAHIAYQBwAGgAaQBjACAAUAByAG8AdgBpAGQAZQByMIID1wYJKoZIhvcNAQcGoIIDyDCCA8QCAQAwggO9BgkqhkiG9w0BBwEwHAYKKoZIhvcNAQwBAzAOBAhCEzpdXXnyawICB9CAggOQjLYlhY1d5ez4DncjxSLoEPaIcEiJrhx04H1vnB4RQWMuAj1BWuBwTSqmK8e/BAZgjTnfuYZ+6oCrKTRCCrIeNb8XZcwfCgUAKXnMww6pbxYF0w48e4rXtkq7KXNiPAyd4CMa0oBUF9QgLTXkifXeQiRc10Ypo853pbRf7vsAMXPtpTGFGc77qFGURRWCRZy3uRBhAMadXi/xOwbHEQcr1m4leyWY56h7G8KjocMTEQ3dkQpD2g4HM9938XUMs/nE8gjwgtCQ0blDlR4kEglL2KxOU5ZyJHxLMJ640wccOjeB9wXc5V5NEYdrK3TsD3P5Zoe9OY8tgzl0iY8zdkHqevjPHMybN42QQiToUry7aBMT5VVUHFlSX3lz3zeHb9th7gwHUY1gjB34lK1ce20rrmJnpbeXD7wN51k20lDA/0cr8qKPnTB5XR//PpQYz0MPzemR1UYiPDIdZSyrGSVMizNCv73mm+4G0RwhBCh/ss6UXFou9M5s+NTi+9lL+N7vC7yAh+UuAzO91iy2OHyk/BV9MldMxYQi48batyXlzvmQFQ1SWdDPdQJWVU2YChQyyL9FkmMH6RNtcVQGMpNqqk/of5/ju7OuLxnZh+cqe1LA0gsAQfjqwzmaH8XQ6AGeO0mzSIgsvNDfXG63EClVwpoDLJ190OL+hYl8n13Mm3eUjvmvzNQXubPGCJhmm0CgdGB0DXgfIQPctZVrmbOT86OfOtfut0vXdjUf3k123PJ+Hg1Uz0YUxTQpaXKRBbML8CCmLlKHzMZHVJcEdvzD50J7WYucH6bhQs74GkOFPrGpD4rrNtbFyayKLlauysx/dYkhyMMsXKJC7ZJJ1jxxAMYUQ1VeqXVSeBDCmsnrUATCrYzGqK+gREOUCDpnuWxYlVQ5KdCfiVDDs4tKG9ELq1l88AgtJHzrSXTIcCwjS5wFI0NicuJbF0253tQXx+85Po/a3NmDjQ0u7ymo1fJgmCJxfvP6CPngk/6r56vwHAV5dFuFvBO6aqs7CASkab9Ki7FisQPnJ9fEaLS26XQE7nELPXkbmIJOU7orKr4GtcwNe7ylT+GY+fQSpNjJWmSUz2occEZwPyhawcBzmpRkp6xDAUa7EzWqx2DgLCt5Ha9DGoU1+tWyi5RyGj6KLiUdu29QAyPeGKOofYK0cwZn353UDr5I7TRLGndi5Tb2GfP9agOy07ZxsW7ONRxTzKV7MDswHzAHBgUrDgMCGgQU13ZXbeRBXBZ0v1GgtcOhTUdxJz8EFF2FIqSzlnXahmLM9HnEghqOjALiAgIH0A==","contentType":"application/x-pkcs12","id":"https://sfrp-cli-kv-000002.vault.azure.net/secrets/sfrp-cli-000003/8cb89e1a819c4bf99e79d0ae841941f7","managed":true,"attributes":{"enabled":true,"nbf":1620859225,"exp":1652395825,"created":1620859825,"updated":1620859825,"recoveryLevel":"Recoverable+Purgeable"},"kid":"https://sfrp-cli-kv-000002.vault.azure.net/keys/sfrp-cli-000003/8cb89e1a819c4bf99e79d0ae841941f7"}'
```

Comment:
Credential Scan is failing with CSCAN-GENERAL0020. 
{{Code}}See test_create_cluster_with_separate_kv.yaml line 910 for the code resulting in match
{{Info}}Found X.509 Certificate Private Key.

Can we suppress this? the cert was only created during the recording of the test and removed afterwards

Reply:
Sure. Check on https://github.com/Azure/azure-cli/blob/680f8339ac010a89d4063566fabc5991abc8a4c2/scripts/ci/credscan/CredScanSuppressions.json